specification/attributes/column_naming_convention.md

```diff
@@ -30,6 +30,7 @@ Naming convention for columns appearing in billing data.
 ## Exceptions
 
 * Identifiers will use the "Id" abbreviation since this is a standard pattern across the industry.
+* Stock Keep Units will use the "SKU" abbreviation since this is a standard pattern across the industry.
```

Comment:
It looks SKU means Stock Keep**ing** Unit.

Reference: https://en.wikipedia.org/wiki/Stock_keeping_unit